packages/eas-cli/graphql-codegen.yml

```diff
@@ -0,0 +1,15 @@
+overwrite: true
+schema: 'http://localhost:3000/--/graphql'
+documents:
```

Comment:
Should this be `https://staging.exp.host/--/graphql`? I think we often develop against staging and not everyone has the backend installed. Or what do you think the process of updating the schemas and types in this repo should be like?